docs/how-to/specify-a-base.rst

```diff
@@ -0,0 +1,58 @@
+.. _how-to-specify-a-base:
+
+Specify a base
+==============
+
+All rocks are layered on top of a software base, which determines compatibility with
+their contents. When you craft a rock, you must specify its base, because it can't
+assume a default base.
+
+Bases are either an :ref:`Ubuntu LTS or interim release
+<explanation-bases-lts-and-interim-bases>`, or the :ref:`bare base
+<explanation-bases-bare-bases>`.
+
+
+Ubuntu LTS and interim bases
+----------------------------
+
+Specify an Ubuntu base with:
+
+.. code-block:: yaml
+    :caption: rockcraft.yaml
+
+    base: ubuntu@<version>
+
+For most rocks, select the latest Ubuntu LTS base, as your rock will benefit from that
+release's 10-year support window.
+
+If instead you need access to the features and packages of an interim Ubuntu base,
+declare the interim release.
+
+
+Unmaintained bases
```

Comment:
I think this section should come after "Bare base". The rationale is that the introduction talks about how rocks can be either 'ubuntu based' or have a 'bare base', so keeping the two sections together makes sense (to me)